### PR TITLE
Release notes 4.5.8

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

File: docs/release_notes_128t_4.5.md [hunk 1]
```diff
@@ -3,6 +3,35 @@ title: 128T 4.5 Release Notes
...
+------
+- **I95-39602 Show peer path status under `show peers hostnames`:** Replaced redundant Router column with `Node`, and added a `Status` column.
+------
+- **I95-39632 Power Saver using stale data:** Resolved an issue with the power saver using stale data after the device-type changes between locked and lockless. 
+------
+- **I95-39698 Alibaba Cloud with enhanced networking sometimes doesn't work:** Resolved an issue between Virtio and RSS causing an unbalanced queue.
+------
+- **I95-39788 Inconsistent services after modifying certain service configs:** Changes made to properly handle service modifications for special services.
+------
+- **I95-39798 Token update and available version update stuck on GPG key access:** The upgrade check has been modified to verify the token update.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_4.5.md:24]: **I95-39798 Version update check may get stuck on GPG key access when using an access token:** The software upgrade version check has been modified to support access tokens."

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_4.5.md [hunk 2]
```diff
@@ -3,6 +3,35 @@ title: 128T 4.5 Release Notes
 sidebar_label: 4.5
 ---
 
+## Release 4.5.8
+
+### Resolved Issues
+
+- **I95-38758 Validate management-over-forwarding and default-route:** A missing default-route prevents the ifcfg from generating. A validation check has been put in place to verify that when management-over-forwarding is enabled, that the default route is set to true.
```
[the remark this answers is not in the document]
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_4.5.md:10]: done

File: docs/release_notes_128t_4.5.md [hunk 3]
```diff
@@ -3,6 +3,35 @@ title: 128T 4.5 Release Notes
 sidebar_label: 4.5
 ---
 
+## Release 4.5.8
+
+### Resolved Issues
+
+- **I95-38758 Validate management-over-forwarding and default-route:** A missing default-route prevents the ifcfg from generating. A validation check has been put in place to verify that when management-over-forwarding is enabled, that the default route is set to true. 
+------
+- **I95-39544 Non-persistent template fields:** Fields filled in by a user are no longer persisted with the template. When a template is re-opened, the fields are blank and ready for new values.
```
[the remark this answers is not in the document]
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_4.5.md:12]: Created a New Features section and added this, along with the suggested edit.

File: docs/release_notes_128t_4.5.md [hunk 4]
```diff
@@ -3,6 +3,35 @@ title: 128T 4.5 Release Notes
...
+------
+- **I95-39580 Reference Candidate Config Values in a Template:** Using the `refconfig` tag in Advanced Mode allows you to reference a value from a running configuration.
```
[the remark this answers is not in the document]
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_4.5.md:14]: Moved to new features and edited.

File: docs/release_notes_128t_4.5.md [hunk 5]
```diff
@@ -3,6 +3,35 @@ title: 128T 4.5 Release Notes
...
+
+- **I95-38758 Validate management-over-forwarding and default-route:** A missing default-route prevents the ifcfg from generating. A validation check has been put in place to verify that when management-over-forwarding is enabled, that the default route is set to true. 
+------
+- **I95-39544 Non-persistent template fields:** Fields filled in by a user are no longer persisted with the template. When a template is re-opened, the fields are blank and ready for new values.
+------
+- **I95-39580 Reference Candidate Config Values in a Template:** Using the `refconfig` tag in Advanced Mode allows you to reference a value from a running configuration. 
+------
+- **I95-39602 Show peer path status under `show peers hostnames`:** Replaced redundant Router column with `Node`, and added a `Status` column.
+------
+- **I95-39632 Power Saver using stale data:** Resolved an issue with the power saver using stale data after the device-type changes between locked and lockless.
```
[the remark this answers is not in the document]
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_4.5.md:18]: edited

File: docs/release_notes_128t_4.5.md [hunk 6]
```diff
@@ -3,6 +3,35 @@ title: 128T 4.5 Release Notes
...
+------
+- **I95-39580 Reference Candidate Config Values in a Template:** Using the `refconfig` tag in Advanced Mode allows you to reference a value from a running configuration. 
+------
+- **I95-39602 Show peer path status under `show peers hostnames`:** Replaced redundant Router column with `Node`, and added a `Status` column.
+------
+- **I95-39632 Power Saver using stale data:** Resolved an issue with the power saver using stale data after the device-type changes between locked and lockless. 
+------
+- **I95-39698 Alibaba Cloud with enhanced networking sometimes doesn't work:** Resolved an issue between Virtio and RSS causing an unbalanced queue.
+------
+- **I95-39788 Inconsistent services after modifying certain service configs:** Changes made to properly handle service modifications for special services.
```
[the remark this answers is not in the document]
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_4.5.md:22]: edited

File: docs/release_notes_128t_4.5.md (hunk 1 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_4.5.md:24]: edited

**Review by MichaelBaj — DISMISSED** [review]

File: docs/about_releases.md [hunk 7]
```diff
@@ -14,10 +14,12 @@ However, issues resolved in `4.3.12`, which was released on 3/12/2021 are not ad
 
 | Version                                          | Release            | GA Date            | End of Maintenance | End of Support     |
 | ------------------------------------------------ | ------------------ | ------------------ | ------------------ | ------------------ |
-| [5.1.1](release_notes_128t_5.1.md#release-511)   | April 12, 2021     |                    | Not Applicable*    | October 15, 2021   |
-| [5.1.0](release_notes_128t_5.1.md#release-5.1)   | March 15, 2021     |                    | April 16, 2021     | October 16, 2021   |
-| [5.0.1](release_notes_128t_5.0.md#release-501)   | April 12, 2021     |                    | Not Applicable*    | October 15, 2021   |
-| [5.0.0](release_notes_128t_5.0.md#release-5.0)   | December 18, 2020  |                    | April 16, 2021     | October 16, 2021   |
+| [5.1.2](release_notes_128t_5.1.md#release-512)   | April 30, 2021     |                    | Not Applicable*    | October 16, 2021   |
```
Comment (jeffmart) [inline docs/about_releases.md:17]: Do we want this in for this release right now? (I expect that 5.1.2 will require its own update.)

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**